src/05-derivatives/02-common-data-types.md

```diff
@@ -11,24 +11,23 @@ Template:
         <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
 ```
 
-Processing in this context means transformations of data that does not change
+Preprocessing in this context means transformations of data that do not change
 the number of dimensions of the input and are not explicitly covered by other
 data types in the specification. Examples:
 
- -  Motion-corrected, temporally denoised, and transformed to MNI space bold files.
- -  Inhomogeneity corrected and skull stripped T1w files.
- -  Motion-corrected DWI files.
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
  -  Time-domain filtered and ICA cleaned EEG data
 
 The `space` keyword is recomended to distinguish files with different underlying
-coordinate systems or registered to different reference maps. The
-`desc` keyword is a general purpose field with freeform values. To distinguish
-between multiple different versions of processing for the same input data the
-`desc` keyword should be used.
+coordinate systems or registered to different reference maps.
+The `desc` keyword is a general purpose field with freeform values.
+To distinguish between multiple different versions of processing for the same
+input data the `desc` keyword should be used.
```

Comment:
The phrasing here is not specific enough. It states that 'space` is recommended (I find that OK), and it states that `desc` simply exists (i.e. not recommended, required or whatever).

>To distinguish between ... keyword should be used.

Is that "should" a "SHOULD" according to [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)? 

Is it required/recommended/suggested (i.e. MUST/SHOULD/MAY) to use either one of `desc` or `space`? What about the existing specification of `proc`?

Reply:
I think that this comment can be ignored in relation to the specific PR; I guess it is better discussed in the top-level  common-derivatives PR.

Reply:
I would add a brief part on what `desc` is an abbreviation for (=description?). This helps the reader with grasping and remembering the entity